docs/source/usermanual.rst

```diff
@@ -1211,8 +1211,6 @@ hipsparseDestroy()
 
```

Comment:
Don't forget to update the copyright at the top

Reply:
This file does not contain a copyright as it is what is used in generating the documentation (I don't think any of the .rst files contain a copyright at the top in any of our libraries)